src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
 public class WorkWithFile {
+    private static final String BUY_WORD = "buy";
+    private static final String SUPPLY_WORD = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToTheFile(toFileName, createReport(readDataFromFile(fromFileName)));
+    }
+
+    private String[] readDataFromFile(String nameOfFile) {
```

Comment:
```suggestion
    private String[] readDataFromFile(String fileName) {
```
Usually, the use of articles is avoided in variable names (a, an, the, of, etc)

Reply:
don't fixed